PowerPointLabs/PowerPointLabs/Utils/TempPath.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Windows;
+
+namespace PowerPointLabs.Utils
+{
+    public class TempPath
+    {
+        private static string _tempTestPath = Path.Combine(Path.GetTempPath(), "PowerPointLabsTest\\");
+
+        public static string GetTempTestFolder()
+        {
+            return _tempTestPath;
+        }
+
+        public static bool IsExistingTempFolder()
+        {
+            return Directory.Exists(_tempTestPath);
+        }
+
+        #region Folder Operations
+  
+        public static void CreateTempTestFolder()
+        {
+            Directory.CreateDirectory(_tempTestPath);
+        }
+
+        public static void DeleteTempTestFolder()
+        {
+            const int waitTime = 1000;
+            var tempFolder = _tempTestPath;
+            var retryCount = 10;
```

Comment:
I tried the codes, `retry and wait` is not very ideal, pls remove it. I guess some items cannot be deleted because MSTest process is still running, but DeleteTempTestFolder() is also part of testing. so `retry and wait` won't work here.